ui/appui-react/src/appui-react/frontstage/FrontstageConfig.ts

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Frontstage
+ */
+
+import { CommonProps } from "@itwin/core-react";
+import { ContentGroup, ContentGroupProvider } from "../content/ContentGroup";
+import { ToolItemDef } from "../shared/ToolItemDef";
+import { PanelConfig } from "../stagepanels/PanelConfig";
+import { WidgetConfig } from "../widgets/WidgetConfig";
+
+/** Configuration from which a frontstage is created.
+ * @public
+ */
+export interface FrontstageConfig extends CommonProps {
+  /** Id for the Frontstage */
+  readonly id: string;
+  /** Tool that is started once the Frontstage is activated */
+  readonly defaultTool: ToolItemDef;
+  /** The Content Group providing the Content Views */
+  readonly contentGroup: ContentGroup | ContentGroupProvider;
+  /** Id of the Content View to be activated initially */
+  readonly defaultContentId?: string;
+  /** Any application data to attach to this Frontstage. */
+  readonly applicationData?: any;
```

Comment:
do we want to bring this fwd to the new api?

Reply:
For now I have removed these properties:
**`FrontstageConfig`**: `defaultContentId`, `applicationData`, `isIModelIndependent`, `defaultTool`.
**`PanelConfig`**: `applicationData`, `header`.